routing.ts

```diff
@@ -765,8 +765,10 @@ namespace jacdac {
                 h.handlePacketOuter(pkt)
             }
         } else if (devId == selfDevice().deviceId) {
-            if (!pkt.isCommand)
+            if (!pkt.isCommand) {
+                // control.dmesg(`invalid echo ${pkt}`)
```

Comment:
@pelikhan if you uncomment this, while in multieditor or in makecode with JD frame, you'll get "invalid echo" message on console for all packets that we send (incl. announce packets); they seem to be coming back at us

Reply:
Can you open a bug? :) I'm going to loose this comment.